cmd/trade.go

```diff
@@ -500,6 +505,21 @@ func runTradeCmd(options inputs) {
 	botConfig = convertDeprecatedBotConfigValues(l, botConfig)
 	l.Infof("Trading %s:%s for %s:%s\n", botConfig.AssetCodeA, botConfig.IssuerA, botConfig.AssetCodeB, botConfig.IssuerB)
 
+	tracker := metrics.MakeMetricsTracker(
+		version,
+		runtime.GOOS,
+		*options.ui,
```

Comment:
since we've changed the logic for `ui` (see comment in metricsTracker) we should pass the `guiVersion` if `*options.ui` is enabled